astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -1307,11 +1306,11 @@ def test_data_out_of_range(parallel, fast_reader, guess):
         else:
             assert len(w) == 3
         for i in range(len(w)):
-            assert f"OverflowError converting to FloatType in column col{i+4}" in str(
+            assert f"OverflowError converting to FloatType in column col{i + 4}" in str(
```

Comment:
Why is this being edited?

Reply:
Thank you for the review @eerovaher, This needs to be reverted. I have mistakenly made these formatting changes. I will revert this back.

Reply:
Resolved